packages/core/ios/Sources/DdSdkImplementation.swift

```diff
@@ -30,6 +30,9 @@ public class DdSdkImplementation: NSObject {
     var webviewMessageEmitter: InternalExtension<WebViewTracking>.AbstractMessageEmitter?
 
     private let jsLongTaskThresholdInSeconds: TimeInterval = 0.1;
+    private let jsRefreshRateEventQueue = DispatchQueue(
+        label: "dd-rn-js-refresh-rate-event-queue",
+        target: .global(qos: .utility))
```

Comment:
We already have a shared queue that makes sure all the SDK modules are on the same one, limiting the risk of concurrency issues: https://github.com/DataDog/dd-sdk-reactnative/blob/develop/packages/core/ios/Sources/QueueHolder.swift

I think we could reuse that one instead of creating a new one, what do you think of it?

Reply:
Done